lib/gradeDialog/grade_dialog.dart

```diff
@@ -0,0 +1,190 @@
+import 'dart:core';
+
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+
+class GradeDialog extends StatefulWidget {
+  const GradeDialog({Key? key}) : super(key: key);
+
+  @override
+  _GradeDialogState createState() => _GradeDialogState();
+}
+
+final List<String?> _list = [
+  '6',
+  '7',
+  '8',
+  '9',
+  '10',
+  '11',
+  '12',
+  'K-5',
+];
+int selectedIndex = -1;
+
+class _GradeDialogState extends State<GradeDialog> {
+  @override
+  Widget build(BuildContext context) {
+    return Dialog(
+      backgroundColor: const Color(0xFFFFFFFF),
+      alignment: Alignment.bottomCenter,
+      insetPadding: const EdgeInsets.fromLTRB(10.0, 10.0, 10.0, 10.0),
+      shape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.all(Radius.circular(35.0)),
+      ),
+      child: SizedBox(
+        height: MediaQuery.of(context).size.height * 0.65,
+        width: MediaQuery.of(context).size.width,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: <Widget>[
+            Container(
+              margin: const EdgeInsets.only(top: 44.0),
+              child: const AutoSizeText(
+                "Select Grade",
+                style: TextStyle(
+                  color: Color(0xFF1353CB),
+                  fontSize: 26.0,
+                  fontWeight: FontWeight.bold,
+                  fontFamily: 'Poppins',
+                ),
+                textAlign: TextAlign.center,
+                maxLines: 1,
+              ),
+            ),
+            Expanded(
+              child: Container(
+                margin: const EdgeInsets.fromLTRB(20.0, 20.0, 20.0, 2.0),
```

Comment:
Whyis there a bottom padding of 2

Reply:
added EdgeInsets.only(left: 20.0, top: 20.0, right: 20.0),